src/Reporter/GithubActionReporter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace FriendsOfTwig\Twigcs\Reporter;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GithubActionReporter implements ReporterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function report(OutputInterface $output, array $violations)
+    {
+        foreach ($violations as $violation) {
+            $output->writeln(sprintf(
+                '::%s file=%s, line=%d, col=%d::%s',
+                strtolower($violation->getSeverityAsString()),
+                $violation->getFilename(),
```

Comment:
this requires escaping chars (properties need more escaping than the final data). See https://github.com/actions/toolkit/blob/5e5e1b7aacba68a53836a34db4a288c3c1c1585b/packages/core/src/command.ts#L80-L94 for the official implementation)